runtime/doc/eval.txt

```diff
@@ -39,7 +39,7 @@ List		An ordered sequence of items, see |List| for details.
 
 Dictionary	An associative, unordered array: Each entry has a key and a
 		value. |Dictionary|
-		Examples:
+		Examples: >
```

Comment:
Might want to get into the habit of always using explicit closing markers (`<`) instead of upstream's Hail Mary, at least for those markers we add ourselves.

(One day we will go through all the docs and sanitize them, but today is not that day.)

Reply:
Really? I like the "feature" of omitting the closing `<` because it lets the `}` motion work more often.

Reply:
Ah, I was not aware of that. Is it just the lack of empty line? (We can add one after the closing tag, of course, for extra cake.)

I've been religious about adding them to Nvim docs I touch, on principle as well as because sometimes that fixes (HTML) rendering issues.

Reply:
> Is it just the lack of empty line? (We can add one after the closing tag, of course, for extra cake.)

that's part of it, but also just the closing `<` is unnecessary most of the time so it's noise.

otoh the `}` motion should probably handle "conceal" intellligently.

> I've been religious about adding them to Nvim docs I touch, on principle as well as because sometimes that fixes (HTML) rendering issues.

parser should handle it except in the middle of a list 😔